flow/Net2.actor.cpp

```diff
@@ -432,6 +432,9 @@ class Connection : public IConnection, ReferenceCounted<Connection> {
 		// Socket settings that have to be set after connect or accept succeeds
 		socket.non_blocking(true);
 		socket.set_option(boost::asio::ip::tcp::no_delay(true));
+#if defined(__unixish__)
+		fcntl(socket.native_handle(), F_SETFD, fcntl(socket.native_handle(), F_GETFD) | FD_CLOEXEC);
+#endif
```

Comment:
I've waffled back and forth on if this should go live in Platform.h somewhere.

Reply:
I'm fairly ambivalent about that, personally. There's certainly some appeal to moving it to platform, but if you left it here that wouldn't bother me.

One question: do we need to do any error checking on these calls?

Reply:
@alexmiller-apple ping

Reply:
Ah, yes, thank you, I totally missed this.

... honestly, if they fail, there's nothing we can meaningfully do.  It's not worth crashing the process over.  We could log that it errored, and aggressively suppress it to try and reduce its spammyness, but there wouldn't really be a meaningful action to take as a result.  I can add one anyway if you'd like.

Reply:
I suppose in the hypothetical scenario where it does fail to do its thing, knowing that it happened may help us to avoid confusion when we see it behaving differently than we expect. I'd be in favor of adding a log event for this with the suppression that you suggested.

Reply:
This has been done.